manifest.json

```diff
@@ -12,8 +12,9 @@
     
       "content_scripts": [
         {
-          "matches": ["*://*.github.com/*/pull/*"],
-          "js": ["codeflow.js"]
+          "matches": [ "*://*.github.com/*/pull/*", "*://*.github.com/*/pulls*"],
+          "js": [ "codeflow.js" ],
+          "run_at": "document_idle"

```

Comment:
>run_at": "document_idle" [](start = 11, length = 24)

When on the pr page it's possible to switch between the conversation tab and the files changed tab. We might have to do a bit more to ensure the changes are applied in that case as well. Can be applied on top of your change.